app/renderer/js/notification/darwin-notifications.ts

```diff
@@ -39,45 +44,45 @@ class DarwinNotification {
 		notification.addEventListener('reply', this.notificationHandler);
 	}
 
-	static requestPermission() {
+	static requestPermission(): void {
 		return; // eslint-disable-line no-useless-return
 	}
 
 	// Override default Notification permission
-	static get permission() {
+	static get permission(): NotificationPermission {
 		return ConfigUtil.getConfigItem('showNotification') ? 'granted' : 'denied';
 	}
 
-	set onreply(handler) {
+	set onreply(handler: ReplyHandler) {
 		replyHandler = handler;
 	}
 
-	get onreply() {
+	get onreply(): ReplyHandler {
 		return replyHandler;
 	}
 
-	set onclick(handler) {
+	set onclick(handler: ClickHandler) {
 		clickHandler = handler;
 	}
 
-	get onclick() {
+	get onclick(): ClickHandler {
 		return clickHandler;
 	}
 
 	// not something that is common or
 	// used by zulip server but added to be
 	// future proff.
-	addEventListener(event, handler) {
+	addEventListener(event: string, handler: ClickHandler | ReplyHandler): void {
 		if (event === 'click') {
-			clickHandler = handler;
+			clickHandler = handler as ClickHandler;
 		}
 
 		if (event === 'reply') {
-			replyHandler = handler;
+			replyHandler = handler as ReplyHandler;
 		}
 	}
 
-	notificationHandler({ response }) {
+	notificationHandler({ response }: { response: string }): void {
```

Comment:
Can be separated as a type outside, but this seems fine too.

Reply:
Yeah, I doing that will make it more readable.